lib/Console/Traits/Parallelization.php

```diff
@@ -64,6 +68,21 @@ protected static function configureParallelization(Command $command): void
         ;
     }
 
+    protected function getParallelExecutableFactory(
+        callable $fetchItems,
+        callable $runSingleCommand,
+        callable $getItemName,
+        string $commandName,
+        InputDefinition $commandDefinition,
+        ErrorHandler $errorHandler
+    ): ParallelExecutorFactory {
+        return ParallelExecutorFactory::create(...func_get_args())
+            ->withSegmentSize($this->getSegmentSize())
+            ->withRunBeforeFirstCommand(Closure::fromCallable([$this, 'runBeforeFirstCommand']))
+            ->withRunAfterLastCommand(Closure::fromCallable([$this, 'runAfterLastCommand']))
+            ->withRunAfterBatch(Closure::fromCallable([$this, 'runAfterBatch']));
```

Comment:
Pimcore 11 is `PHP ^8.1`, so why not use [First-class Callable Syntax](https://www.php.net/manual/en/functions.first_class_callable_syntax.php) here:

```suggestion
            ->withRunBeforeFirstCommand($this->runBeforeFirstCommand(...))
            ->withRunAfterLastCommand($this->runAfterLastCommand(...))
            ->withRunAfterBatch($this->runAfterBatch(...));
```